subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/suite/TestParser.kt

```diff
@@ -52,53 +70,39 @@ internal fun parseTest(testInApk: TestInApk, deviceName: DeviceName): TestStatic
     kind = determineKind(testInApk.annotations),
 
     flakiness = testInApk.annotations
-        .find { it.name == FLAKY_NAME }
-        ?.getStringValue(FLAKY_REASON_KEY)
-        ?.let { reason -> Flakiness.Flaky(reason)}
+        .find { it.name == Flaky::class.java.canonicalName }
+        ?.let { Flakiness.Flaky(it.getStringValue(FLAKY_REASON_KEY) ?: NO_REASON) } // todo we can't parse annotations default from dex
```

Comment:
So the bug was because we didn't parse annotation default value and this logic skipped annotation without value
I will create a task for support annotation's default value parsing